src/partials/languages/zh-TW.html

```diff
@@ -22,32 +22,54 @@
 
 <!-- Translations: Chinese (Taiwanese) -->
 {% macro t(key) %}{{ {
-  "language": "zh-Hant",
+  "language": "zh-TW",
   "action.edit": "編輯此頁",
   "action.skip": "跳轉到",
+  "action.view": "查看此頁原始碼",
   "announce.dismiss": "不再顯示此訊息",
+  "blog.archive": "封存",
+  "blog.categories": "分類",
+  "blog.categories.in": "於",
+  "blog.continue": "繼續閱讀",
+  "blog.draft": "草稿",
+  "blog.index": "回到主頁",
+  "blog.meta": "元數據",
+  "blog.references": "相關連結",
   "clipboard.copy": "複製",
   "clipboard.copied": "已複製",
   "consent.accept": "同意",
   "consent.manage": "管理設定",
   "consent.reject": "拒絕",
+  "footer": "頁腳",
   "footer.next": "下一頁",
   "footer.previous": "上一頁",
+  "header": "頁首",
   "meta.comments": "留言",
   "meta.source": "來源",
-  "search.config.lang": "ja",
```

Comment:
Please leave these three settings here, as described in our [contributing guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md#missing-translations).